assets/css/style.css

```diff
@@ -85,3 +85,12 @@ footer {
     width: 300px;
     height: 600px;
   }
+
+table, tr, td, th {
```

Comment:
This is a bit aggressive of a styling, as it is directly transforming a structural element (table) with a display element. Since table could be used in other contexts, would be better to have this as something like:

```
.post-body table {
  ...
}
```